opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -970,6 +971,28 @@ func resourceOpennebulaVirtualMachineUpdate(d *schema.ResourceData, meta interfa
 		}
 	}
 
+	if d.HasChange("context") {
```

Comment:
You also should update Provider doc as this is only possible from release 5.10 of OpenNebula.
Chaning context on oldest release won't have effect

Reply:
The updateconf method seems a bit older than that.
It seems to be introduced by this commit: https://github.com/OpenNebula/one/commit/32e15605cd22d402fd9a3ba4126034925e9b0c75

From version 5.0 of OpenNebula